### ISSUE TITLE
UI - SubscriptionTestCase.test_positive_upload_and_delete - elements have changed

### ISSUE THREAD

**Issue by rplevka** [SatelliteQE/robottelo#4733]
There's a change introduced in the Manifest dialog.
The `delete manifest` button now no longer contains `ui-sref` attribute that we used to use for locating the element.

Also, an additional confirming dialog has been introduced while deleting a manifest.

```html
<button class="btn btn-default" ng-hide="denied('delete_manifest')" ng-click="openModal()">
         
         <span ng-show="deleteTask.pending" translate="" class="ng-scope ng-hide"><span class="ng-scope">Deleting Manifest...</span></span><span bst-modal="deleteManifest()" template-url="subscriptions/manifest/views/manifest-delete-modal.html" class="ng-isolate-scope"></span><i class="fa fa-spinner fa-spin ng-hide" ng-show="deleteTask.pending"></i>
          <!-- translate:  -->
          <!-- translate:  --><span ng-hide="deleteTask.pending" translate="" class="ng-scope"><span class="ng-scope">Delete Manifest</span></span>
      </button>
```

**Comment by abalakh** [comment]
Changes introduced by following BZ: https://bugzilla.redhat.com/show_bug.cgi?id=1266827

**Comment by abalakh** [comment]
Addressed in #4789, #4795, #4740 , #4807
